modules/abstract-cosmos/src/lib/transactionBuilderFactory.ts

```diff
@@ -1,20 +1,24 @@
 import { BaseTransactionBuilderFactory, InvalidTransactionError, TransactionType } from '@bitgo/sdk-core';
-import { OsmoTransactionBuilder } from './transactionBuilder';
-import { OsmoTransferBuilder } from './transferBuilder';
 import { BaseCoin as CoinConfig } from '@bitgo/statics';
-import { OsmoTransaction } from './transaction';
 import { StakingActivateBuilder } from './StakingActivateBuilder';
 import { StakingDeactivateBuilder } from './StakingDeactivateBuilder';
 import { StakingWithdrawRewardsBuilder } from './StakingWithdrawRewardsBuilder';
+import { CosmosTransaction } from './transaction';
+import { CosmosTransactionBuilder } from './transactionBuilder';
+import { CosmosTransferBuilder } from './transferBuilder';
+import { CosmosUtils } from './utils';
 
 export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
-  constructor(_coinConfig: Readonly<CoinConfig>) {
+  protected utils: CosmosUtils;
+
+  constructor(_coinConfig: Readonly<CoinConfig>, utils: CosmosUtils) {
```

Comment:
I think this would cause problem, is it possible to create utils instance here based on the coinConfig?